stl/inc/xstring

```diff
@@ -2432,6 +2438,33 @@ public:
         _Proxy._Release();
     }
 
+    explicit basic_string(_String_constructor_concat_tag, const basic_string& _Alsource, const _Elem* const _Left_ptr,
+        const size_type _Left_size, const _Elem* const _Right_ptr, const size_type _Right_size)
+        : _Mypair(
+            _One_then_variadic_args_t(), _Alty_traits::select_on_container_copy_construction(_Alsource._Getal())) {

```

Comment:
We're currently inconsistent, but in new code, I'd like to use `{}` empty braces for tags like `_One_then_variadic_args_t`. I'll file an issue about cleaning up our sources; I don't think we have one yet.

Reply:
Do you want me to fix up the tags in this file or just this one? (I'm not a fan of making it inconsistent with all the other ctors over this but if we're at least consistent within a file that'd be OK)

Reply:
I filed #468 for the STL-wide overhaul, so I think I'd prefer to see parens used here for consistency and avoiding making this PR bigger.